### ISSUE TITLE
Set entrypoint to aws

### ISSUE THREAD

**Issue by shnhrrsn** [mikesir87/aws-cli-docker#9]
Any chance you can update to use the entrypoint as aws?  It'd be helpful to use with whalebrew which requires it: https://github.com/bfirsh/whalebrew#creating-packages

I'm currently using mesosphere/aws-cli with whalebrew, but it’s not kept up to date and I’d like to switch :)

Thanks!

**Comment by mikesir87** [comment]
Hi @shnhrrsn! This was actually a deliberate decision to _not_ make aws as the entrypoint. The reasoning was to make it easier to do workflows in CI builds where you might need to get data, use `jq` to change things, and then send it back to AWS.

However, I think it would be awesome to support whalebrew. One thought I have is to actually turn the Dockerfile into a multi-stage build in which one stage sets the entrypoint. Then, I can update the build to produce two images. Something like this...

- `latest`, `1.16.113`, etc. - work as is
- `latest-e`, `1.16.113-e`, etc. - have the entrypoint set.

Basically, any image tags with a `-e` suffix have the entrypoint set. Thoughts?
